0002_child_test.js

```diff
@@ -378,32 +383,31 @@ describe("Test Social Worker Note:", () => {
       cy.log(
          "Placeholder things to make test complete: if the popup close is fixed these will need to be removed."
       );
-      cy.get(".webix_button")
-         .filter(":visible")
-         .contains("Close")
-         .click({ force: true, multiple: true });
-      cy.get(
-         cyInterfaceCARS.page.socialWorker.page.children.view.child.page
-            .socialWork.page.notes.form.addNote.field.categories
-      )
-         .find(".webix_multicombo_input")
-         .click();
+      // cy.get(".webix_button")
```

Comment:
Same with these, delete the un-needed code :D

Reply:
OK Isaac, Thank You